### ISSUE TITLE
Add RRFS network to obsproc's capability

### ISSUE THREAD

**Issue by ilianagenkova** [NOAA-EMC/obsproc#74]
To address RRFS model's observations needs, add a RRFS network to obsproc's capability, and generate desired bufr dump and prepbufr obs files. 

**Due December 2022**
- [x] Leverage off RAP network's scripts, ush scripts and triggers to set RRFS up and running (rrfs, rrfs_e, rrfs_p)
- [x] Update RRFS to obsproc.v1.1.0 to generate gmi1cr, snocvr, saldrn and subpfl, add Sent-6 to gpsro, add Metop-B/C AMV to satwnd, add saldrn and subpfl to nsstbufr
- [x] Split the large radiances dump group to speed up processing (for global, and may be later for rrfs)
- [x] Replace -np xx with -np $NPROCS (see exrap_dump.sh as a good example) (IG)
- [x] Replace mpiexec with launcher_DUMP (see exurma_dump.sh as a good example) (IG)
- [x] Resolve BUFRLIB: CLOSBF warnings in dump jobs(http://www2.spa.ncep.noaa.gov/bugzilla/show_bug.cgi?id=1350) (IG - _under testing_)
- [x] Unify time dump window - set _earliest  to -1h, and _latest to +1h (confirm with @JacobCarley-NOAA ) ( /Utils/GO_CHKDAT (?)/go_chkdat dump_file_name)
- [x] Generate CONUS 15min AMVs dumps (in separate "regional" satwnd dumps) (bufr-dump/fix/bufr_dumplist , *satwhr*.bufr_d,  exrrfs_dump*.sh : 00505**) (SN - _under testing & soon to add to Ashley's collecting branch_
- [x] Verify and maintain generation of Direct Broadcast radiances  (es*** and ***db bufr dumps)
- [x] Confirm all GPSRO observation streams - DO5, Eumetsat, any others?
- [x]  Confirm domain - see mask files

Add JPSSS-2 **NOAA-21** products of interest. **Due at NOAA-21 provisional readiness, early 2023**
- [x]  L1B ATMS rads (IG - _tanks are generated in NRT, need to start dumping_)
- [x]  L1B CrISS rads (IG - _tanks are generated in NRT, need to start dumping_
- [x] VIIRS (or AVHRR??) AMVs - (IG - _waiting on NESDIS_ )
- [ ] SST - (IG - _waiting on NESDIS_ )
- [ ] Land surface temperature - _Need info from RRFS team , Waiting on NESDIS_

**Due March 2023** (May-June 2023 is more realistic)
- [x] Add UPRAIR full resolution bufr dumps (need Chris Hill's T2B code to be completed)
- [x] Add T2B prepbufr (need Chris Hill's T2B code to be completed)
       ~~~The next few products are *nc and will be ingested by DataFlow, tickets submitted or soon to be submitted (Matt P.)
- [ x] Add VIIRSS Fire radiative power (RAVE product sample due March 2023)
- [ x] Add GOES Fire radiative power (also in RAVE??)
- [ x] Add RAVE (uses VIIRSS add GOES Fire radiative power as inputs, see emails Shobha, chat with Ravan)
- [ x] Add AOD and ADP aerosol products - on PDA, format? Satingest or DF? see email Amanda
- [ x] Add surface pm2.5 air constituents , smoke and dust NESDIS products - EPA AirNow - email Amanda and Hongli 
- [ x] Add GOES Lightning Mapper data (source data, format, routing, etc) -  Amanda Black, NESDIS products, submit ticket to DF  (data on LDM) see email with Amanda
      ~~~
- [x] Add wind tower data - review tanks and update bufr_dump to include dumping it (bufr_dumplist, *wndtow* 002/020, 002/021, 000/021, exrrfs_dump.sh, add to smallest Dump group (least run time) )

**Comment by ilianagenkova** [comment]
Development Branch: [https://github.com/NOAA-EMC/obsproc/tree/feature/RRFS.obsproc.addition](https://github.com/NOAA-EMC/obsproc/tree/feature/RRFS.obsproc.addition)

**Comment by ilianagenkova** [comment]
Wind Tower tanks:
export TANK_000020=/lfs/h1/ops/dev/dcom/202304*/b000/xx020 (wind energy nacelle)
export TANK_002020=/lfs/h1/ops/dev/dcom/202304*/b002/xx020 (wind energy tower)
export TANK_002021=/lfs/h1/ops/dev/dcom/202304*/b002/xx021 (wind energy tower)
Will not go into prepbufr, due to BUFR table differences.

**Comment by ilianagenkova** [comment]
Change jrrfs_*.wc2.pbs triggers :
From:
#PBS -A OBSPROC-DEV
#PBS -q dev 

To:
#PBS -A RRFS-DEV
#PBS -q pridev

Group: emc.lam
Group space: /lfs/h2/emc/lam/noscrub/emc.lam

**Comment by ilianagenkova** [comment]
@ShunLiu-NOAA , this was the user request as to what to go into the RRFS observational network. 

I will close this ticket, as we met all requirements' (from @JacobCarley-NOAA and @MatthewPyle-NOAA ). RRFS is going into obsproc v1.3 in production. 

If the RRFS obs files need to be updated/refined, we can work on it with another issue.

**Comment by MatthewPyle-NOAA** [comment]
Thanks @ilianagenkova .  Is there a known timeline for obsproc v1.3 going into production?

**Comment by ilianagenkova** [comment]
Late 2024 - early 2025

**Comment by ShunLiu-NOAA** [comment]
Great!
The timeline is good for RRFS implementation.

Thank you.

Shun
